src/main/java/org/candlepin/dto/manifest/v1/SubscriptionDTO.java

```diff
@@ -125,6 +125,15 @@ public String getOwnerId() {
         return this.getOwner() != null ? this.getOwner().getId() : null;
     }
 
+    /**
+     * @return the key to the owner associated with this subscription
+     */
+    @Override
```

Comment:
For methods we override, we can save some documentation pain/overhead by just inheriting the docs:

```
/**
 * {@inheritDoc}
 */
```

Unless our implementation does something weird or special, we shouldn't bother rewriting the docs and having to keep multiple places in sync.

(Note: this occurs in several places)

Reply:
Updated throughout